### PR TITLE
Ning

### PR DESCRIPTION
**Pull request by nngrz**

### CODE REVIEW

File: plates.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+# no period, spaces, or punctuation
+def is_punctuation(s):
+    disallowed_c = set(string.punctuation + " ")
+    for n in s:
+        if n in disallowed_c:
+            return False
+    return True
+
+main()
```
Comment (balinskia) [inline plates.py:56]: There is a new line at the end. See the red icon in the file below

File: plates.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    # print("is_length:", is_length(s))
+    # print("is_middle_letter", is_middle_letter(s))
+    # print("is_first_number", is_first_number(s))
+    # print("is_punctuation", is_punctuation(s))
+    if  (is_first_two_letter(s) and
+        is_length(s) and
+        is_middle_letter(s) and 
+        is_first_number(s) and
+        is_punctuation(s)):
+        return True
```
Comment (balinskia) [inline plates.py]: See "the same comment for your pull request" in doc

**Review by balinskia — CHANGES_REQUESTED** [review]

File: plates.py (hunk 1 above)
Comment (nngrz) [reply plates.py:56]: indeed, i can only see the new line in twttr file, but not in plates file

File: plates.py (hunk 2 above)
Comment (nngrz) [reply plates.py]: check now

File: twttr.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+def main():    
+    output = ""
+    for c in s:
+        if not is_vowel(c):
+            output += c
+    print("Output:", output)
+
+def is_vowel(c):
+    if c in vowels:
```
Comment (balinskia) [inline twttr.py]: The same comment as above

File: twttr.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+vowels = ["A", "a", "E", "e", "I", "i", "O", "o", "U", "u"]
+
+output = input("Input: ")
+
+for v in vowels:
+    output = output.replace(v, "")
+
+print("Output:", output)
+'''
```
Comment (balinskia) [inline twttr.py]: Add new line

File: plates.py (hunk 2 above)
Comment (balinskia) [reply plates.py]: There is no change here. Please refactor

**Review by balinskia — CHANGES_REQUESTED** [review]

**Comment by balinskia** [comment]
Lgtm

**Review by balinskia — APPROVED** [review]

**Pull request merged**